data-prepper-plugins/cwl-sink/.gitignore

```diff
@@ -0,0 +1,42 @@
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
```

Comment:
This file should removed. I think it was not intended.

Reply:
Those files should be removed now.